pkg/network/ddnpmapi.h

```diff
@@ -240,6 +240,7 @@ typedef struct filterPacketHeader
     uint64_t        pktSize;                //! size of packet
     uint64_t        af;		                //! address family of packet
     uint64_t        ownerPid;               //! (-1) if not available
+    uint64_t        timestamp;              // timestamp in ns since unix epoch
```

Comment:
Unused?

Reply:
This will be used, and was the reason for the update.

Reply:
it will be; that was the whole point of the driver PR.